src/Compilers/Core/Portable/Syntax/ChildSyntaxList.cs

```diff
@@ -175,7 +177,7 @@ internal static SyntaxNodeOrToken ChildThatContainsPosition(SyntaxNode node, int
             // The targetPosition must already be within this node
             Debug.Assert(node.FullSpan.Contains(targetPosition));
 
-            var green = node.Green;
+            GreenNode? green = node.Green;
```

Comment:
what effect does this have now that 'var' is always inferred to have a nullable type?

Reply:
Once the `var` change is fully in our toolset this won't be necessary. But I was seeing warnings for this, possibly because of VS code, hence I went with the explicit type here.